### PR TITLE
rtpbreak: add mirror to fix broken download

### LINKED ISSUE #36558: Broken link in formula: rtpbreak
$ brew install rtpbreak 
```
==> Downloading http://dallachiesa.com/code/rtpbreak/rtpbreak-1.3a.tgz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "rtpbreak"
Download failed: http://dallachiesa.com/code/rtpbreak/rtpbreak-1.3a.tgz
```

The original ```rtpbreak``` project on ```sourceforge.net``` has no files in it.

I have found a tarball on ```packetstorm.net``` but it doesn't compile; can someone restore a compiling source tarball for this formula.

### PR DESCRIPTION
**Pull request by DomT4**
Closes #36558.

### CODE REVIEW

File: Library/Formula/rtpbreak.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
-require 'formula'
-
 class Rtpbreak < Formula
-  homepage 'http://www.dallachiesa.com/code/rtpbreak/doc/rtpbreak_en.html'
-  url 'http://dallachiesa.com/code/rtpbreak/rtpbreak-1.3a.tgz'
-  sha1 'd22e9c37cc28c2fc36475d221b4eb2cc2c5aafbb'
+  # Homepage and URL dead since at least Feb 2015
+  homepage "http://www.dallachiesa.com/code/rtpbreak/doc/rtpbreak_en.html"
+  url "http://dallachiesa.com/code/rtpbreak/rtpbreak-1.3a.tgz"
+  mirror "https://raw.githubusercontent.com/DomT4/LibreMirror/master/Rtpbreak/rtpbreak-1.3a.tgz"
```
Comment (mikemcquaid) [inline Library/Formula/rtpbreak.rb]: How did you get the mirror, just out of interest?

**Comment by mikemcquaid** [comment]
Is there a `make install` we can use?

File: Library/Formula/rtpbreak.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/rtpbreak.rb]: Found a working link on a reasonably well-known security tools website, downloaded the ` .tgz `, checksummed it to check it matched the expected contents, and then stuck it on my mirror when it came back as a match.

**Comment by DomT4** [comment]
Actually, there is. I wonder why we weren't using that before. Will see if using it causes any problems locally.

**Comment by DomT4** [comment]
Pushed to use ` make install `. It's an unusual prefix handling, which might be why it wasn't used before.

**Comment by mikemcquaid** [comment]
Thanks @DomT4, great work!

**Comment by DomT4** [comment]
Thanks Mike.

**Pull request closed without merging**